### ISSUE TITLE
Extend FLOSS to use the rendering techniques pioneered by QUANTUMSTRAND

### ISSUE THREAD

**Issue by mr-tz** [mandiant/flare-floss#943]
[QUANTUMSTRAND](https://github.com/mandiant/flare-floss/tree/quantumstrand/floss/qs) is an experiment that augments traditional strings.exe output with context to aid in malware analysis and reverse engineering. For example, we show the structure of a file alongside its strings and mute/highlight entries based on their global prevalence, library association, expert rules, and more.

FLOSS should use the techniques pioneered by QUANTUMSTRAND to highlight important information while muting common and/or analytically irrelevant noise.

**Comment by Anu-Ra-g** [comment]
@mr-tz Can I setup floss on WSL2? Or flare-vm is preferred?

**Comment by williballenthin** [comment]
no preference, as long as the environment is comfortable for you to develop Python

**Comment by andy0712dik** [comment]
@mr-tz i would like to contribute to this project because it is looking interesting I am final-year student and my interest is in cybersecurity can you please guide me that where should i start to get involve in this project

**Comment by mr-tz** [comment]
A good start would be to familiarize yourself with the tool, documentation, and source. Then you can browse the open issue list and/or ask focused questions.

**Comment by officialasishkumar** [comment]
Is there any slack or element channel?

**Comment by danishzulfiqar** [comment]
Can you assign issue to me, I can start getting understanding of it

**Comment by williballenthin** [comment]
@danishzulfiqar no, this is a large-scoped issue tracking a project candidate for GSoC'24. If you'd like to contribute there, please follow our posted guidance on how to get started.
